util/settings/settings.go

```diff
@@ -66,9 +66,11 @@ type ArgoCDSettings struct {
 	// Secrets holds all secrets in argocd-secret as a map[string]string
 	Secrets map[string]string `json:"secrets,omitempty"`
 	// KustomizeBuildOptions is a string of kustomize build parameters
-	KustomizeBuildOptions string
+	KustomizeBuildOptions string `json:"kustomizeBuildOptions,omitempty"`
```

Comment:
Did this fix a bug?

Reply:
No AFAIK, but saw it and did it as housekeeping.